packages/core/src/domain/session/sessionStore.ts

```diff
@@ -123,9 +131,30 @@ export function startSessionStore<TrackingType extends string>(
     return sessionState
   }
 
+  function initializeSession() {
+    processSessionStoreOperations(
+      {
+        process: (sessionState) => {
+          if (!isSessionStarted(sessionState)) {
+            return getInitialSessionState(sessionState)
+          }
+        },
+        after: (sessionState) => {
+          sessionCache = sessionState
+        },
+      },
+      sessionStoreStrategy
+    )
+  }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏if this operation pass the session from not started to started, should we name it `startSession` instead?
and similarly reinitialize -> restart